app/src/main/java/comcmput301f16t01/github/carrier/Request.java

```diff
@@ -66,7 +66,9 @@ public int getStatus() {
     }
 
     public void setStatus(int status) {
+        this.status = status;
         // TODO make sure you do this right - Mandy (i.e. check that the status can change from one state to another)
+        // TODO make an actual test for this (Mandy)
```

Comment:
Most of our tests need revision... :P I may slowly work on all request tests next week...